unity-renderer/Assets/Scripts/MainScripts/DCL/Environment/Environment.cs

```diff
@@ -93,6 +93,7 @@ public class Model
             public readonly PlatformContext platform;
             public readonly WorldRuntimeContext world;
             public readonly HUDContext hud;
+            public bool IsBeingDestroyed { get ; private set; }
```

Comment:
`Environment` shouldn't be storing this kind of state. `DataStore_Common` is a better alternative.

Reply:
This is the idea I was looking for!